### PR TITLE
Update use of `b0.71` CI image name for this branch

### PR DESCRIPTION
**Pull request by portante**

### CODE REVIEW

**Review by webbnh — APPROVED** [review]
👍

File: jenkins/branch.name [hunk 1]
```diff
@@ -1 +1 @@
-main
+b0.71
```
[the remark this answers is not in the document]
Comment (portante) [reply jenkins/branch.name:1]: See PR #2788.

**Review by ndokos — DISMISSED** [review]

**Review by webbnh — APPROVED** [review]
This looks like scope-creep, but...what the heck, it's only four lines....

**Review by portante — COMMENTED** [review]
These small changes are much easier to verify in one PR with multiple commits than to make them all in separate commits and try to see that everything is working by the end.

**Review by dbutenhof — APPROVED** [review]

**Pull request merged**